### PR TITLE
sUSDe

### PR DESCRIPTION
**Pull request by avniculae**

### CODE REVIEW

File: packages/tomls/src/omnibus/reya_network.toml [hunk 1]
```diff
@@ -586,3 +610,15 @@ cp1Rusd_usdeParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address %>
 cp1Rusd_usdeParentConfig_priceHaircutUnscaled = "0.075"
 cp1Rusd_usdeParentConfig_autoExchangeDiscountUnscaled = "0.01"
 cp1Rusd_usdeParentConfig_oracleNodeId = "<%= settings.usdeUsdcNodeId %>"
+# -----------------------------------------------
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_network.toml:613]: need to decrease cap of USDe and wETH above

File: packages/tomls/src/omnibus/reya_network.toml [hunk 2]
```diff
@@ -203,6 +212,11 @@ wbtcOptimismWithdrawFeeUnscaled = "0.0000022"
 wbtcPolygonWithdrawFeeUnscaled = "0.0000011"
 # -------------------------------------------------------------------
 usdeEthereumWithdrawFeeUnscaled = "27.888"
+# -------------------------------------------------------------------
+susdeEthereumWithdrawFeeUnscaled = "27.888"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_network.toml:216]: double check values against new avarages sent by socket

Comment (avniculae) [reply packages/tomls/src/omnibus/reya_network.toml:216]: (for all tokens)

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 3]
```diff
@@ -529,7 +551,7 @@ market11Uni_marketOrderMaxStaleDuration = "0"
 
 [var.cp_1rusd]
 cp1Rusd_maxMarkets = "11"
-cp1Rusd_maxCollaterals = "4"
+cp1Rusd_maxCollaterals = "5"
```
Comment (ioanaroceanu2000) [inline packages/tomls/src/omnibus/reya_cronos.toml:554]: why so many?

File: packages/tomls/src/omnibus/reya_network.toml [hunk 4]
```diff
@@ -510,7 +534,7 @@ market11Uni_marketOrderMaxStaleDuration = "0"
 
 [var.cp_1rusd]
 cp1Rusd_maxMarkets = "11"
-cp1Rusd_maxCollaterals = "3"
+cp1Rusd_maxCollaterals = "4"
```
Comment (ioanaroceanu2000) [inline packages/tomls/src/omnibus/reya_network.toml:537]: weth, rusd, susde, usde right?

File: packages/tomls/src/omnibus/testnet_source_chains.toml [hunk 5]
```diff
@@ -16,6 +20,7 @@ tokensRouterPackage = "reya-tokens:1.0.0@router"
 [var.initial_proxy_vars]
 wstethSalt = "wsteth-main"
 usdeSalt = "usde-main"
+susdeSalt = "susde-main"
```
Comment (ioanaroceanu2000) [inline packages/tomls/src/omnibus/testnet_source_chains.toml:23]: testnet this is susde-main-2

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 3 above)
Comment (avniculae) [reply packages/tomls/src/omnibus/reya_cronos.toml:554]: rusd, weth, wstETH, usde, susde

File: packages/tomls/src/omnibus/reya_network.toml (hunk 4 above)
Comment (avniculae) [reply packages/tomls/src/omnibus/reya_network.toml:537]: right

File: packages/tomls/src/omnibus/testnet_source_chains.toml (hunk 5 above)
Comment (avniculae) [reply packages/tomls/src/omnibus/testnet_source_chains.toml:23]: this is for source chains. i already deployed them with this salt and used them in the bridge, so will keep it like this for now

**Pull request merged**